src/main/java/io/github/moulberry/notenoughupdates/NEUOverlay.java

```diff
@@ -1135,7 +1135,7 @@ public void consume(int x, int y, int id) {
 				}
 				if (internalname.get() != null) {
 					if (itemstack.get() != null) {
-						if (NotEnoughUpdates.INSTANCE.config.hidden.enableItemEditing && Keyboard.getEventCharacter() == 'k') {
+						if (NotEnoughUpdates.INSTANCE.config.apiKey.repositoryEditing && Keyboard.getEventCharacter() == 'k') {
```

Comment:
why is it called apiKey here and the class is called api data??

Reply:
because otherwise everyone has to run /api new again

Reply:
i could write a bunch of code to change that, but i didnt feel like it

Reply:
idc, make them run api new

Reply:
no

Reply:
:angry:

Reply:
fine ill write the stupid code